app/src/main/res/layout-land/reset_pin_dialog.xml

```diff
@@ -10,21 +10,31 @@
 
   <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:profile="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
-    <org.oppia.android.app.profile.ProfileInputView
-      android:id="@+id/input_pin"
+    <com.google.android.material.textfield.TextInputLayout
+      android:id="@+id/reset_pin_input_pin"
+      style="@style/TextInputLayout"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:layout_marginStart="24dp"
       android:layout_marginTop="32dp"
       android:layout_marginEnd="24dp"
       android:layout_marginBottom="20dp"
-      app:inputLength="3"
-      app:isPasswordInput="true"
-      app:label="Administrator's PIN"
-      profile:error="@{viewModel.errorMessage}" />
+      android:hint="@{@string/admin_settings_enter_user_new_pin(viewModel.name)}"
```

Comment:
could you confirm is it working correctly and not giving issues in orientation changes?

Reply:
@anandwana001 Yes this works correctly Added a GIF for reference

<img src="https://user-images.githubusercontent.com/54740946/104270047-962cc280-54bd-11eb-9444-971f3a39c405.gif" width="33%"/>

Reply:
let's merge it.

Reply:
Before the merge, please confirm the test cases too on espresso, for robolectric, it's running on GitHub action so we will rely on it.

Reply:
I've run the tests I'll add screenshot in a hour

Reply:
@anandwana001 pls check
![image](https://user-images.githubusercontent.com/54740946/104275491-f412d780-54c8-11eb-9125-d29c409c5f1b.png)